### ISSUE TITLE
print precision and recall

### ISSUE THREAD

**Issue by banyous** [ibrahimsharaf/doc2vec#21]
First of all I would like to thank you for sharing with us this nice project,
I am getting accuracy and F1 score when running the code,
How I can get precision an recall?
thanks

**Comment by ibrahimsharaf** [comment]
Hello @banyous, thanks! 

You are welcome to create a new pull request adding them, you can use them from sklearn.
- https://scikit-learn.org/stable/modules/generated/sklearn.metrics.precision_score.html
- https://scikit-learn.org/stable/modules/generated/sklearn.metrics.recall_score.html

**Comment by ibrahimsharaf** [comment]
Resolved https://github.com/ibrahimsharaf/doc2vec/pull/22
